chains/smartbch/metadata.yaml

```
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://smartscout.cash/api
+    family: blockscout
+    name: smartbchscan
+    url: https://smartscout.cash
+blocks:
+  confirmations: 1
+  estimateBlockTime: 3
+  reorgPeriod: 15
+chainId: 10000
+deployer:
+  name: Goblins Cash
+  url: https://goblins.cash
+displayName: Smartbch
+domainId: 10000
+isTestnet: false
+name: smartbch
+nativeToken:
+  decimals: 18
+  name: BCH
+  symbol: BCH
+protocol: ethereum
+rpcUrls:
+  - http: https://smartbch.greyh.at
+  - http: https://rpc.smartbch.org
+  - http: https://global.uat.cash
```

third rpc doesn't work
```suggestion
rpcUrls:
  - http: https://smartbch.greyh.at
  - http: https://rpc.smartbch.org
```